mpt_extension_sdk/runtime/djapp/conf/default.py

```diff
@@ -213,6 +214,10 @@
     os.getenv("MPT_ORDERS_API_POLLING_INTERVAL_SECS", "120")
 )
 
+MPT_NOTIFY_CATEGORIES = json.loads(
```

Comment:
Could you mark here a TODO to unify the way how do we handle this in extensions. The thing is that today, it is parsed in runtime in `initializer`. @robcsegal is moving it to SDK now